src/safeposix/syscalls/sys_calls.rs

```diff
@@ -43,8 +43,6 @@ use crate::safeposix::filesystem::{decref_dir, metawalk, Inode, FS_METADATA};
 use crate::safeposix::net::NET_METADATA;
 use crate::safeposix::shm::SHM_METADATA;
 
-
-
 impl Cage {
```

Comment:
Changes for this file are only the formatting changes done using `cargo fmt`. No new changes have been made in this PR.